ecs-cli/modules/cli/servicediscovery/servicediscovery_app_test.go

```diff
@@ -389,6 +389,102 @@ func TestCreateServiceDiscoveryWithECSParamsExistingPublicNamespaceByName(t *tes
 	testCreateServiceDiscovery(t, "awsvpc", input, emptyContext(), validateNamespace, validateSDS, false)
 }
 
+func TestUpdateServiceDiscovery(t *testing.T) {
```

Comment:
for both Update and Delete SD, could you add a test for the error case(s), like if the stack name can't be found, or update/deleteStack fails?